operator_manifest/helpers.py

```diff
@@ -72,4 +73,13 @@ def get_error_message(error):
                     path or 'at top level', error.validator,
                     ", ".join(error_contexts) or error.message)
 
-    return error_message
\ No newline at end of file
+    return error_message
+
+
+def sha256sum(s, abbrev_len=0, prefix=False):
+    prefix_ = 'sha256:' if prefix else ''
+    data = s.encode() if isinstance(s, str) else s
+    digest = hashlib.sha256(data).hexdigest()
+    if abbrev_len > 0:
+        digest = digest[0:abbrev_len]
+    return prefix_ + digest
```

Comment:
newline

Reply:
I am pretty sure that flake is handling those new lines, I can rebase those flake8 fixes to the original commits, but that would not be pleasant.